src/main/java/io/reactivex/Observable.java

```diff
@@ -317,15 +317,17 @@ public static int bufferSize() {
         return lift(new OperatorTake<>(n));
     }
     
-    public <U> Observable<T> takeUntil(Publisher<U> other) {
+    public final <U> Observable<T> takeUntil(Publisher<U> other) {
```

Comment:
See #1353 for concerns about the parameter name "other" and suggestions for a more consistent parameter naming scheme.